<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWindow</class>
 <widget class="QMainWindow" name="DashboardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel Principal - System Medic Query</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_estados">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestión de Estados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pacientes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestión de Pacientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_usuarios">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestión de Usuarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ingresos">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestión de Ingresos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_egresos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestión de Egresos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reportes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
